ironic_tempest_plugin/tests/scenario/ironic_standalone/test_basic_ops.py

```diff
@@ -141,8 +141,8 @@ def test_ip_access_to_server(self):
 class BaremetalDriverDirectWholedisk(bsm.BaremetalStandaloneScenarioTest):
 
     api_microversion = '1.31'  # to set the deploy_interface
-    if 'redfish' in CONF.baremetal.enabled_hardware_types:
-        driver = 'redfish'
+    if 'idrac' in CONF.baremetal.enabled_hardware_types:
```

Comment:
Revert this to original code. If this is a valid scenario for with idrac as driver, create a new class and test for the same.